screens/courses/ViewCourseScreen.js

```diff
@@ -120,11 +157,30 @@ class ViewCourseScreen extends React.Component {
     });
```

Comment:
I prefer doing:
```javascript
<Text key={index}>{`Session ${index + 1}: ${ session.weekday }'s ${ start } to ${ end }`}</Text>
```
in order to avoid all this nasty highlighting